tools/loader/ImageClassifier.cpp

```diff
@@ -232,11 +254,11 @@ template <typename ElemTy> static void applySoftmax(Handle<ElemTy> H) {
   }
 }
 
-/// Given the output Softmax Tensor \p SMT and \p functionName, print the
-/// results of inference.
+/// Given the output Softmax Tensor \p SMT and \p functionName, prints the
+/// results of inference and returns number of incorrect predictions
```

Comment:
nit: Add backslash to returns for Doxygen style (i.e. `\returns`).

Reply:
Fixed in the last commit